docs/content.zh/docs/dev/table/functions/systemFunctions.md

```diff
@@ -24,161 +24,161 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# System (Built-in) Functions
+# 系统（内置）函数
 
-Flink Table API & SQL provides users with a set of built-in functions for data transformations. This page gives a brief overview of them.
-If a function that you need is not supported yet, you can implement a [user-defined function]({{< ref "docs/dev/table/functions/udfs" >}}).
-If you think that the function is general enough, please <a href="https://issues.apache.org/jira/secure/CreateIssue!default.jspa">open a Jira issue</a> for it with a detailed description.
+Flink Table API & SQL 为用户提供了一组内置的数据转换函数。本页简要介绍了它们。如果您需要的功能尚不支持，您可以实现
+[用户自定义功能]({{< ref "docs/dev/table/functions/udfs" >}})。如果你觉得这个功能够通用， 请
```

Comment:
there is no doubt that the `function` is translated into `函数`.
but I don't agree that the author wants to express `不够通用`, because only when the function is general enough you can open a Jira for it.

Reply:
Good correction. So sorry for twisting the meaning.